<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectDialog</class>
 <widget class="QDialog" name="ConnectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="serverIP">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>服务器IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serverPort">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>端口</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userPassword">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QComboBox" name="serverType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>MySQL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SQLite3</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="databaseName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>数据库名</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
